release-1.6/release-notes-draft.md

```diff
@@ -58,12 +58,59 @@
 
 ## Cluster Lifecycle
 ### New Features
+- kubeadm is a simple tool for creating Kubernetes clusters on existing
+  machines. kubeadm's command line interface is now in beta.  Using it is
+  considered safe, although note that upgrades and HA are not yet supported.
+  Please [try it out](https://kubernetes.io/docs/getting-started-guides/kubeadm/) and
+  [give us feedback](https://kubernetes.io/docs/getting-started-guides/kubeadm/#feedback)!
+- Debian packages are additionally built for ppc64le and s390x architectures.
+- kubeadm now deploys etcd3, and no longer supports etcd2.
+
 ### Notable Changes
-- New Bootstrap Token authentication and management method. Works well with kubeadm.
-- Adds a new cloud-controller-manager binary that may be used for testing the new out-of-core cloudprovider flow
-- Introduces an API for the Kubelet to request TLS certificates from the API server.
-- kubeadm is enhanced and improved with a baseline feature set and command line flags that are now marked as beta.
+- kubeadm 1.6 now sets up a more secure cluster by default.  It uses
+  [RBAC](https://kubernetes.io/) to grant limited privileges to workloads
+  running on the cluster.  By default, service accounts and users are
+  restricted from accessing the API. They can be granted permissions
+  [using the RBAC command-line utilities](https://kubernetes.io/docs/admin/authorization/rbac/#command-line-utilities).
```

Comment:
https://kubernetes.io/docs/admin/authorization/rbac/#command-line-utilities does not exist